bin/i18n/sync-out.rb

```diff
@@ -181,6 +181,8 @@ def restore_redacted_files
           plugins << 'vocabularyDefinition'
         elsif original_path.starts_with? "i18n/locales/original/curriculum_content"
           plugins.push(*Services::I18n::CurriculumSyncUtils::REDACT_RESTORE_PLUGINS)
+        elsif File.fnmatch('i18n/locales/original/blockly-mooc/*lab.json', original_path)
+          plugins << 'link'
```

Comment:
There are links to redact in lab i18n content or is this just to be sure?

Reply:
The answer to your question is no, in fact only a couple labs have links. 
But the underlying idea for the wild card is that all *lab.json would contain same type of strings and links can be added to those files. And whenever new labs are added (like musiclab), no additional work is needed to include them in the sync.

Reply:
I removed other labs that were already included in the i18n pipeline. Otherwise some existing strings in crowdin would have been modified.